Sample test `test_streaming_automl_action_recognition` failing with `The Model does not exist`

See build log [here](https://source.cloud.google.com/results/invocations/91613946-c663-4e48-8ae3-69aa7099f908/log) and stack trace below.
```
___________________ test_streaming_automl_action_recognition ___________________

capsys = <_pytest.capture.CaptureFixture object at 0x7fac64ff1ad0>
video_path = '/tmp/pytest-of-kbuilder/pytest-0/video0/file.mp4'

    @pytest.mark.flaky(max_runs=3, min_passes=1)
    def test_streaming_automl_action_recognition(capsys, video_path):
        project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
        model_id = "6597443600609443840"
        beta_snippets.streaming_automl_action_recognition(video_path, project_id, model_id)
        out, _ = capsys.readouterr()
>       assert "segment" in out
E       AssertionError: assert 'segment' in 'com.google.cloud.ai.platform.common.errors.AiPlatformException: code=NOT_FOUND, message=The Model does not exist., cause=null\n'

```

--- a/samples/analyze/beta_snippets_test.py
+++ b/samples/analyze/beta_snippets_test.py
@@ -166,7 +166,7 @@ def test_streaming_automl_object_tracking(capsys, video_path):
 @pytest.mark.flaky(max_runs=3, min_passes=1)
 def test_streaming_automl_action_recognition(capsys, video_path):
     project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
-    model_id = "6597443600609443840"
+    model_id = "6652522536091254784"
     beta_snippets.streaming_automl_action_recognition(video_path, project_id, model_id)
     out, _ = capsys.readouterr()
     assert "segment" in out

